README.md

```diff
@@ -159,11 +155,15 @@ Download packaged binaries from the [releases page][].
 
 ### Build from source
 
-See here on how to [build GitHub CLI from source](/docs/source.md).
+See here on how to [build GitHub CLI from source][].
 
-[docs]: https://cli.github.com/manual
+[official docs]: https://cli.github.com/manual
 [scoop]: https://scoop.sh
 [Chocolatey]: https://chocolatey.org
 [releases page]: https://github.com/cli/cli/releases/latest
 [hub]: https://github.com/github/hub
 [contributing page]: https://github.com/cli/cli/blob/trunk/.github/CONTRIBUTING.md
+[issue]: https://github.com/cli/cli/issues/new/choose
+[more detailed explanation]: /docs/gh-vs-hub.md
+[community repo]: https://www.archlinux.org/packages/community/x86_64/github-cli/
```

Comment:
I'm not sure if the phrases `more detailed explanation` and `community repo` are really clear when viewed out of context in this link section. Could we use a more descriptive markdown link alias here, perhaps `gh-vs-hub` instead of `more detailed explanation` and `arch linux repo` instead of `community repo`? The link text visible to users can stay the same.